### PR TITLE
Add `description` and `concurrent_build_limit` to `aws_codebuild_project`

### PR DESCRIPTION
**Pull request by gui-don**
## what
* allows user to input a concurrent_build_limit for the codebuild project
* allows user to input a description for the codebuild project

## why
* Missing and needed

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -16,6 +16,17 @@ variable "environment_variables" {
   description = "A list of maps, that contain the keys 'name', 'value', and 'type' to be used as additional environment variables for the build. Valid types are 'PLAINTEXT', 'PARAMETER_STORE', or 'SECRETS_MANAGER'"
 }
 
+variable "description" {
+  default     = ""
```
Comment (SweetOps) [inline variables.tf:20]:
```suggestion
  default     = "Managed by Terraform"
```
let's define some value

**Review by SweetOps — CHANGES_REQUESTED** [review]

File: variables.tf [hunk 2]
```diff
@@ -16,6 +16,17 @@ variable "environment_variables" {
   description = "A list of maps, that contain the keys 'name', 'value', and 'type' to be used as additional environment variables for the build. Valid types are 'PLAINTEXT', 'PARAMETER_STORE', or 'SECRETS_MANAGER'"
 }
 
+variable "description" {
+  default     = "Managed by Terraform"
```
Comment (nitrocode) [inline variables.tf:20]:
```suggestion
  type        = string
  default     = "Managed by Terraform"
```

**Review by nitrocode — APPROVED** [review]

**Pull request merged**